packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -118,15 +128,31 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
     if (!attached) {
       return;
     }
+
+    _stopwatch.reset();
+    _stopwatch.start();
```

Comment:
Hmm, I wonder if this could be implemented so that it doesn't run by default, it's probably not very heavy, but still?

Reply:
Sure, I can add a boolean called `trackPerformance` to control this.

Reply:
The `if` will probably be more expensive than actually running it, so an if is probably not an option :sweat_smile:
I'm not sure if there is any solution, since it can't be made in to a mixin or such...

Reply:
One way I can think of is to add `ValueChange` callbacks for update and render. Then based on `trackPerformance` we can swap the callbacks with the normal or tracked versions.

```
```

Reply:
Then we also don't add code in the core which would be preferable

Reply:
Okay, let me try that 👍🏼

Reply:
That works exactly the same! Thanks for suggesting the mixin approach 🙂

Reply:
Super, looks good! 🙂